### DISCUSSION TITLE
[Feature] Allow basic authentication in Server Settings > External Domain

### DISCUSSION THREAD

**Discussion by volkovd** [immich-app/immich#13456] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Previously, I was able to add basic authentication in Server Settings > External Domain. However, after clicking save, the basic authentication is now being automatically stripped. This feature was very useful because all shared links automatically contained the basic authentication string. The basic authentication is provided by the Nginx reverse proxy, adding an extra layer of security for the Immich server exposed to the internet. 
![image](https://github.com/user-attachments/assets/861801b5-b58f-4af1-8bc8-f00b74e9b6e4)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
#15230
